interfaces/mount/entry_test.go

```diff
@@ -161,4 +161,10 @@ func (s *entrySuite) TestOptsToFlags(c *C) {
 	c.Assert(flags, Equals, syscall.MS_RDONLY|syscall.MS_NODEV|syscall.MS_NOSUID)
 	_, err = mount.OptsToFlags([]string{"bogus"})
 	c.Assert(err, ErrorMatches, `unsupported mount option: "bogus"`)
+	// The x-snapd-prefix is reserved for non-kernel parameters that do not
+	// translate to kernel level mount flags. This is similar to systemd or
+	// udisks use fstab options to convey additional data.
```

Comment:
s/udisks use/udisks that use/

Reply:
Done in af52b4fe78eb0a2db88f8813332cbad6ddb07358